src/main/scala/com/persona/service/chat/dao/ChatDAO.scala

```diff
@@ -0,0 +1,55 @@
+package com.persona.service.chat.dao
+
+import java.util.UUID
+
+import slick.driver.PostgresDriver.api._
+
+import com.persona.service.chat.ChatMessage
+
+case class MsgHistory(
+                       msgId: Int,
+                       offerId: UUID,
+                       userId: String,
+                       msgType: Int,
+                       sender: String,
+                       message: String
+                     )
+
+class MsgHistoryTable(tag: Tag) extends Table[MsgHistory](tag, "msg_history") {
+
+  def msgId = column[Int]("msg_id", O.AutoInc)
+  def offerId = column[UUID]("offerid")
+  def userId = column[String]("userid")
+  def msgType = column[Int]("type")
+  def sender = column[String]("sender")
+  def message = column[String]("message")
+
+  def idx = index("idx_msghistory", (offerId, userId))
+
+  override def * = (msgId, offerId, userId, msgType, sender, message) <> (MsgHistory.tupled, MsgHistory.unapply)
+
+}
+
+object ChatDAO extends TableQuery(new MsgHistoryTable((_))) {
+
+  var instance: Option[Database] = None;
```

Comment:
Why not make ChatDAO a class and pass the db in it's constructor?  See com.persona.service.account.SlickAccountDAO